explorer/site-config/hca-dcp/dev/projectViewModelBuilder.ts

```diff
@@ -36,19 +37,9 @@ export const projectToContacts = (
 
 export const projectsToContributors = (
```

Comment:
Looking at the naming convention here; does it make sense to use a `buildX` format here in the view builder, and then a `getX` format in the corresponding transformer? It looks like we have `getProjectContacts` and `getProjectDescription` in the `projectTransformer` already, and maybe the `buildX` terminology makes more sense here in the builder? What are your thoughts?